### ISSUE TITLE
Polygon type is not supported

### ISSUE THREAD

**Issue by neokree** [synw/geojson#1]
Hello, 

I have tried this library, but it seems that a `NoSuchMethodError` exception is thrown if the `type` of the geometry object is set to `Polygon`.
Looking at the code it seems that the only two supported types at this time are:
- MultiPolygon
- LineString

But the [RFC spec](https://tools.ietf.org/html/rfc7946#section-1.4) states that there are seven:
"Point", "MultiPoint", "LineString",
      "MultiLineString", "Polygon", "MultiPolygon", and
      "GeometryCollection"

**GeoJson used**
```
{"type":"FeatureCollection","features":[{"type":"Feature","properties":{},"geometry":{"type":"Polygon","coordinates":[[[9.183021763511533,45.28718757700191],[9.184502342887754,45.28750460520285],[9.186154583640928,45.28422856168324],[9.184706190772886,45.28394925889815]]]}}]}
```
geojson validated with http://geojson.io

Please add a doc that state the current supported types and use an exception more meaningful (or do not return values for unsupported types, your choose)

**Comment by Karlheinzniebuhr** [comment]
Same issue, getting this error when using GeoJson of type Polygon
`NoSuchMethodError (NoSuchMethodError: The getter 'iterator' was called on null.`

**Comment by synw** [comment]
The first implementation was weak, sorry. I published a 0.2.1 with major changes and it now supports polygons and all the other feature types except GeometryCollection. Let me know if it works now. I still have to add exceptions as suggested by @neokree  and better error handling
